packages/synapse-react-client/src/components/QueryVisualizationWrapper.tsx

```diff
@@ -139,15 +134,8 @@ export function QueryVisualizationWrapper(
       }))
     }
   }, [isFacetsAvailable])
-
-  useEffect(() => {
-    if (isLoadingNewBundle) {
-      setSelectedRowIndices([])
-    }
-  }, [isLoadingNewBundle])
```

Comment:
Location of this state is going to be interesting for cohort builder. Might need to move it elsewhere (maybe in a new context provider that wraps QueryVisualizationWrapper?), but that can be figured out later.